src/vcpkg/versions.cpp

```diff
@@ -1,8 +1,34 @@
+#include <vcpkg/base/messages.h>
 #include <vcpkg/base/parse.h>
+#include <vcpkg/base/stringview.h>
 #include <vcpkg/base/util.h>
 
 #include <vcpkg/versions.h>
 
+namespace
+{
+    using namespace vcpkg;
+
+    DECLARE_AND_REGISTER_MESSAGE(VersionInvalidRelaxed,
+                                 (msg::version),
+                                 "",
+                                 "Error: String `{version}` is not a valid Relaxed version string (semver with "
```

Comment:
```suggestion
                                 "string `{version}` is not a valid relaxed version string (semver with "
```

you should use `msg::msgErrorMessage` (or w/e we call it)